randovania/server/multiplayer/world_api.py

```diff
@@ -174,42 +209,30 @@ def world_sync(sio: ServerApp, request: ServerSyncRequest) -> ServerSyncResponse
     sessions_to_update_meta = set()
 
     for uid, world_request in request.worlds.items():
-        try:
+        with sentry_sdk.start_transaction(op="message", name="sync_one_world") as span:
+            span.set_tag("world_id", uid)
+            span.set_data("message.error", 0)
```

Comment:
L235 sets it anyways to `span.set_data("message.error", failed_syncs.get(uid, 0))`. It's redudant?

Reply:
Paranoia with making sure it's set even if some random exception happens.